src/System.IO.Pipes/src/Resources/Strings.resx

```diff
@@ -120,12 +120,18 @@
   <data name="ArgumentOutOfRange_NeedNonNegNum" xml:space="preserve">
     <value>Non negative number is required.</value>
   </data>
+  <data name="ArgumentOutOfRange_NeedValidPipeAccessRights" xml:space="preserve">
+    <value>Need valid PipeAccessRights value.</value>
```

Comment:
Nit, `Invalid PipeAccessRights value.` ?

Reply:
Yes it is better. I just ported it from IO.Pipes.AccessControl. Will update